src/tfr_navigation/package.xml

```diff
@@ -4,20 +4,19 @@
   <version>0.0.0</version>
   <description>The tfr_navigation package</description>
 
-  <!-- If you are the actual maintainer of this package, please say so here. -->
-  <maintainer email="ryan.berge@trickfirerobotics.com">Ryan Berge</maintainer>
   <license>BSD</license>
+
+  <maintainer email="ryan.berge@trickfirerobotics.com">Ryan Berge</maintainer>
```

Comment:
Why is Ryan the maintainer?

Reply:
It just got auto generated when ryan set up the repository. There is a card in the trello asking us to put some general trickfire email down at some point in all of our packages.

Reply:
It actually wasn't auto-generated; I just figured it was better having some email than no email.

Reply:
I think catkin will also do a build warning if you don't have this tag filled in anyway.